### ISSUE TITLE
Animations not working

### ISSUE THREAD

**Issue by fabianluque** [pleerock/ng2-modal#22]
I noticed in my app that animations are not currently working. I checked in the samples you provide and they don't work there either.

Are animations supported currently?

**Comment by pleerock** [comment]
yes animations aren't supported yet. would be great if you can contribute into it
